setup.py

```diff
@@ -24,7 +24,7 @@ def has_ext_modules(foo) -> bool:
 
 
 setuptools.setup(name='cohere',
-                 version='3.7.0',
+                 version='4.0.0',
```

Comment:
version doesn't match up with  CHANGELOG.md (which has 3.8)

Reply:
also note #160 is going to 4.0 as well so please coordinate